internal/handlers/handlers.go

```diff
@@ -1 +1,78 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func MainHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		http.Error(w, fmt.Sprintf("Method %s not allowed", r.Method), http.StatusMethodNotAllowed)
+		return
+	}
+
+	html, err := os.ReadFile("index.html")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	_, err = w.Write(html)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "POST" {
+		http.Error(w, fmt.Sprintf("Method %s not allowed", r.Method), http.StatusMethodNotAllowed)
+		return
+	}
+
+	file, _, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Это проблема клиента
```suggestion
		http.Error(w, err.Error(), http.StatusBadRequest)
```